dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/FieldSymbol.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2019 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.google.cloud.tools.opensource.classpath;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+import java.util.Objects;
+
+/** Symbol for a field of a class. */
+final class FieldSymbol extends Symbol {
+  private final String name;
+  private final String descriptor;
+
+  FieldSymbol(String className, String name, String descriptor) {
```

Comment:
Does it work to store just the single binary name of each symbol, rather than splitting it into className and member name?

Reply:
It would work. However, the caller have to combine them when it creates this instance and the user of the data later splits the field into className and field.